rust/auction/program/src/processor/create_auction.rs

```diff
@@ -44,6 +44,8 @@ pub struct CreateAuctionArgs {
     pub tick_size: Option<u64>,
     /// Add a minimum percentage increase each bid must meet.
     pub gap_tick_size_percentage: Option<u8>,
+    /// Add a instant sale price.
+    pub instant_sale_price: Option<u64>,
```

Comment:
Again, changing an existing contract struct. This will break older UIs. The best way to go with this one is to actually create a new contract endpoint called create_auction_v2.rs that takes a modified struct and just calls the original with more parameters, something like this, so you have complete code re-use but one takes a slightly different struct.